src/components/SplitField/SplitField.jsx

```diff
@@ -1,45 +1,70 @@
-import { forwardRef } from 'react';
-import { Button, Card, FormControl, InputGroup } from 'react-bootstrap';
-import { faTrashCan } from '@fortawesome/free-regular-svg-icons';
-import { faPlusMinus } from '@fortawesome/free-solid-svg-icons';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { EXACT_AMOUNTS, PERCENTAGES, SHARES } from '../../constants';
+import { forwardRef, useState, useContext } from "react";
+import { Button, Card, FormControl, InputGroup } from "react-bootstrap";
+import { faTrashCan } from "@fortawesome/free-regular-svg-icons";
+import { faPlusMinus } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { EQUALLY, EXACT_AMOUNTS, PERCENTAGES, SHARES } from "../../constants";
+import SplitDataContext from "../../store/split-data-context";
 
-import './styles.scss';
+import "./styles.scss";
 
 export const SplitField = forwardRef((props, ref) => {
+  const splitState = useContext(SplitDataContext);
+
+  const [exactAmtInput, setExactAmtInput] = useState("");
```

Comment:
I think `exactAmount` would be a better name here since adding "Input" does little to add to its description and there's less ambiguity since "Amount" isn't abbreviated
```suggestion
  const [exactAmount, setExactAmount] = useState("");
```